app/src/main/java/org/gem/indo/dooit/views/custom/NonSwipableViewPager.java

```diff
@@ -0,0 +1,57 @@
+package org.gem.indo.dooit.views.custom;
```

Comment:
Scrolling is disabled on API 17 with and without the scroller. @rjacobs31 Is the scroller working as intended on API 19?

Scrolling is enabled on API 24 regardless.